### PR TITLE
Add "contains" function tests

### PR DESCRIPTION
**Pull request by efeichen**

### CODE REVIEW

**Comment by efeichen** [comment]
Tests for `contains( )` Mavo function (PR [#349](https://github.com/mavoweb/mavo/pull/349)).

`contains(null, null)` returns false, which seems weird to me but is consistant with search(null, null) which returns -1.

**Comment by LeaVerou** [comment]
Kenneth, after looking at your tests I've also realized something else: To make the more complex cases possible while still returning an array of booleans for arrays, it means `contains()` needs to handle that internally. I.e. if `contains()` was what called `contains()` and it got an array of booleans back, just pass it through `Mavo.Functions.or()` to get back a single boolean.

@karger, thoughts? Please look at Kenneth's tests before you reply (they are easy to read).

File: functions.html [hunk 1]
```diff
@@ -407,6 +407,53 @@ <h1>String slicing</h1>
...
+            <td>
+                contains(["Kenneth", "MIT", {name: "Kenneth", food: "rice", quantity: 6}, "sleeping"], {name: "Kenneth", work: "MIT"})
+            </td>
+            <td>[false,false,false,false]</td>
+        </tr>
+        <tr>
+            <td>
+                contains(null, null)
+            </td>
+            <td>false</td>
```
Comment (LeaVerou) [inline functions.html]: I'd also suggest:

`contains("foo", null)`
`contains(null, "foo")`
`contains()`
`contains("foo")`
`contains([], "foo")`
`contains("0", 0)`

**Comment by efeichen** [comment]
All the tests are now passed because I edited the `...a.slice(b.length)` to `					...a.slice(b.length).map((n) => o.scalar(o.identity, n))` so that it does operation with the identity.

Meanwhile, the `and()` test that failed is still failing. This PR is for the contains() tests, so I don't know if I should edit that test so it shows that it passes.

**Pull request merged**